pkg/flag/kubernetes_flags.go

```diff
@@ -79,6 +79,13 @@ var (
 		Default:    "trivy-temp",
 		Usage:      "specify the namespace in which the node-collector job should be deployed",
 	}
+	ExcludeOwned = Flag{
+		Name:       "exclude-owned",
+		ConfigName: "kubernetes.exclude.owned",
+		Shorthand:  "E",
```

Comment:
We have a lot of flags now and want to assign shorthand to frequently used flags.

Reply:
@chen-keinan Your thoughts on this?